deployer/templates/fluentd.yaml

```diff
@@ -159,6 +159,10 @@ objects:
               value: ${USE_MUX_CLIENT}
             - name: "MUX_ALLOW_EXTERNAL"
               value: ${MUX_ALLOW_EXTERNAL}
+            - name: "BUFFER_QUEUE_LIMIT"
```

Comment:
We still use this template for CI testing, unless @stevekuznetsov is going to get rid of that.  So it needs to be in both places.